lib/obs_github_deployments/cli/commands/is_locked.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ObsGithubDeployments
+  module CLI
+    module Commands
+      class IsLocked < Dry::CLI::Command
+        desc "Checks if the deployment is locked"
+
+        argument :repository, required: true, desc: "The GitHub repository to check for locked"
+        argument :token, required: true, desc: "The required GitHub access token"
+
+        def call(repository:, token:)
+          puts client(repository: repository, token: token).locked?
```

Comment:
can you remove the puts? We should call it in the executable, not here.

Reply:
Done

Reply:
Looks like the `puts` in the main script returns something different from the value returned in this `call`. I have to revert this until I find something better